Kitodo/src/main/webapp/pages/login.xhtml

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
Is this line really needed? Other file did not have this line or did they miss this line?